src/mmw/mmw/settings/base.py

```diff
@@ -358,6 +359,9 @@ def get_env_setting(setting):
     'authorize_url': 'auth/concord_id/authorize',
     'access_token_url': 'auth/concord_id/access_token',
     'user_json_url': 'auth/concord_id/user.json',
+
```

Comment:
This blank line added to differentiate the ITSI Portal specific settings, and MMW specific settings (which follow).